resolwe_bio/processes/workflows/rnaseq.yml

```diff
@@ -229,3 +229,103 @@
           mode: union
           stranded: '{{input.stranded}}'
           id_attribute: '{{input.id_attribute}}'
+
+- slug: workflow-meta-rnaseq
```

Comment:
Call this dss-rnaseq. You can read about decision support systems here: https://en.wikipedia.org/wiki/Decision_support_system. Instead of a `dss` prefix, maybe we could use `dsc` (decision support component) or `dsp` (decision support process).